indykite/import_utils.go

```diff
@@ -35,9 +36,36 @@ func basicStateImporter(_ context.Context, data *schema.ResourceData, _ any) ([]
 }
 
 func parseImportID(d *schema.ResourceData) error {
-	if gidBase64Regex.MatchString(d.Id()) {
+	importID := d.Id()
+
+	// Check if it's a GID (direct ID import)
+	if gidBase64Regex.MatchString(importID) {
 		return nil
 	}
 
-	return errors.New("Unimplemented id format: " + d.Id())
+	// Check if it's a name with location parameter (e.g., "my-resource?location=gid:xxx")
+	// The ID will be kept as-is and used in the read function
+	if strings.Contains(importID, "?location=") {
+		return nil
+	}
+
+	return errors.New("Unimplemented id format: " + importID +
+		". Expected either 'gid:xxx' or 'resource-name?location=gid:xxx'")
+}
```

Comment:
Question: Have you tested this really works? I'm not sure this is supported in Terraform. To import something by query. Because importer is setting ID internally.